### PR TITLE
Move to different versioning scheme and show version in About dialog

### PR DESCRIPTION
**Pull request by chirayudesai**
For Android 10

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,8 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="com.stevesoltys.seedvault"
-    android:versionCode="7"
-    android:versionName="1.0.0">
+    android:versionCode="29000001"
+    android:versionName="10-1.0">
```
Comment (chirayudesai) [inline app/src/main/AndroidManifest.xml:6]: 2 lines changed, from 30 and 11 respectively.

Comment (chirayudesai) [reply app/src/main/AndroidManifest.xml:6]: @stevesoltys last release was 1.0.0, guessing this one could be 10-1.0 ?

We should bump this given the changes we've done though.

Comment (stevesoltys) [reply app/src/main/AndroidManifest.xml:6]: If we are putting out a release for the backported changes, `10-1.1.0` sounds good to me. I think we should keep version bumps for releases in separate PRs though.

**Review by stevesoltys — DISMISSED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (stevesoltys) [reply app/src/main/AndroidManifest.xml:6]: So to answer your question.. maybe `10-1.0.0` for this PR.

**Review by grote — APPROVED** [review]

**Comment by chirayudesai** [comment]
Looks like a travis failure,

> The command "~/bin/install-jdk.sh --target "/home/travis/openjdk11" --workspace "/home/travis/.cache/install-jdk" --feature "11" --license "GPL" --cacerts" failed and exited with 7 during .

How do we make it retry?

**Review by grote — DISMISSED** [review]

**Comment by chirayudesai** [comment]
@stevesoltys I think we'll just need a CI override at this point. I can't trigger a retry, if you can maybe that helps, otherwise we can just merge.

**Comment by stevesoltys** [comment]
Yeah was gonna say, can just force push the commit again. All good now @chirayudesai?

**Comment by chirayudesai** [comment]
> Yeah was gonna say, can just force push the commit again. All good now @chirayudesai?

Yeah, all good, just needs approval and merging. I made the version `10-1.0.0`, and I think we can move to an org, and then tag and release that, and follow soon after for 11.

**Review by stevesoltys — APPROVED** [review]

**Pull request merged**